app/models/room.rb

```diff
@@ -54,6 +55,12 @@ def change_skills_by(lang_ids = [])
     end
   end
 
+  def evaluations_score(output = nil, round = nil)
+    return output if evaluations.blank?
+    score = evaluations.sum {|ev| ev.score } / evaluations.length
```

Comment:
__[RuboCop]__

Style/SymbolProc: Pass `&:score` as an argument to `sum` instead of a block.


<!-- Comment by [Sider](https://sider.review) -->